scenarioo-client/app/diffViewer/types/StepDescription.ts

```diff
@@ -0,0 +1,21 @@
+import {ILabels, IStepDescription} from '../../generated-types/backend-types';
+import {StepDiffInfo} from './StepDiffInfo';
+
+export class StepDescription {
+    details: { [index: string]: any };
```

Comment:
do we really need to map to any here?

Reply:
This is defined in this way in the backend-types as well:
```
export interface IDetailable {
    details: { [index: string]: any };
}
```